warehouse/query-core/src/main/java/datawave/core/iterators/DatawaveFieldIndexCachingIteratorJexl.java

```diff
@@ -326,68 +330,67 @@ public DatawaveFieldIndexCachingIteratorJexl() {
         this.timeFilter = null;
         this.datatypeFilter = null;
         
-        this.fs = null;
+        this.ivaratorCacheDirs = null;
+        this.controlFs = null;
+        this.controlDir = null;
         this.queryLock = null;
-        this.uniqueDir = null;
         this.allowDirReuse = false;
         this.scanThreshold = 10000;
         this.hdfsBackedSetBufferSize = 10000;
         this.maxOpenFiles = 100;
+        this.numRetries = 2;
         this.maxRangeSplit = 11;
         
         this.sortedUIDs = true;
     }
     
     protected DatawaveFieldIndexCachingIteratorJexl(Builder builder) {
```

Comment:
seems like builder can have a variety of accepted forms, can you add javadoc to formalize these?

Reply:
Sure.  Are you saying that you want me to just list the various extensions of the builder here?

Reply:
Added some comments.  Good to go?